src/tetris/scene/menuScene.ts

```diff
@@ -40,9 +51,14 @@ export default class MenuScene extends Phaser.Scene {
 	private _optionsButton: TextButton;
 	private readonly _game: Game;
 	private _pipeline: Phaser.Renderer.WebGL.Pipelines.TextureTintPipeline;
+	private _localPlayerName: String;
 	//endregion
 
 	//region private methods
+	private async _setLocalPlayerName(): Promise<void> {
+		this._localPlayerName = (document.querySelector('#playername') as any).value;
```

Comment:
Why not use `as HTMLElement` or `as HTMLInputElement`?

Reply:
Nice suggestion, will do :)